src/imio/smartweb/core/upgrades/configure.zcml

```diff
@@ -479,4 +487,19 @@
     />
   </genericsetup:upgradeSteps>
 
+  <genericsetup:upgradeSteps
+      source="1035"
+      destination="1036"
+      profile="imio.smartweb.core:default">
+    <genericsetup:upgradeDepends
+        title="Add new content type : imio.smartweb.CirkwiView"
+        import_profile="imio.smartweb.core.upgrades:upgrade_1035_to_1036"
+    />
+    <genericsetup:upgradeDepends
+        title="Reload types info to add imio.smartweb.CirkwiView"
+        import_profile="imio.smartweb.core:default"
+        import_steps="typeinfo"
```

Comment:
This is not needed because you already add types related changes in `upgrade_1035_to_1036` profile.

Reply:
Only `types.xml` was missing --> upgrade step was not working.